integration-tests/bats/sql.bats

```diff
@@ -792,6 +792,51 @@ SQL
     rm -rf doltsql
 }
 
+@test "sql: batch delimiter" {
```

Comment:
Make sure this works for delimiters embedded in strings, e.g.

```
mysql> delimiter //
mysql> insert into mytable values (4, "//")
    -> //
Query OK, 1 row affected (0.01 sec)
```

Reply:
If it doesn't work for this use case (not clear if the quote matching code above is good enough), then fine to add a skipped test instead

Reply:
Ohh that's a good idea!